.github/workflows/republish_released_docker_containers.yml

```diff
@@ -39,6 +39,21 @@ jobs:
 
   build:
     runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        # Split up images to publish so that longer ones are able to run independently/finish faster, otherwise this takes >6 hours
+        # Any task which is skipped from a broader task must be explicitly included in this list to avoid accidentally missing new
+        # tasks as they are added.
+        images_to_publish: [
+          {"gradle_task": ":pushAllRunnersDockerImages", "skip_flags": ""},
+          {"gradle_task": ":sdks:python:container:push39", "skip_flags": ""},
+          {"gradle_task": ":sdks:python:container:push310", "skip_flags": ""},
+          {"gradle_task": ":sdks:python:container:push311", "skip_flags": ""},
+          {"gradle_task": ":sdks:python:container:pushAll", "skip_flags": "-Pskip-python-39-images -Pskip-python-310-images -Pskip-python-311-images"},
```

Comment:
do we not have a push312 command here?

Reply:
We do, but I intentionally didn't use it to avoid a situation where we add python 3.13 but forget to update this file. With the current setup, we will still push 3.13, the workflow will just take longer.